src/partials/toc-item.html

```diff
@@ -28,7 +28,7 @@
 
   <!-- Table of contents list -->
   {% if toc_item.children %}
-    <nav class="md-nav" aria-label="{{ toc_item.title }}">
+    <nav class="md-nav" aria-label="{{ toc_item.title|striptags }}">
```

Comment:
Please stick to our code style and insert spaces left and right to the `|` pipe ☺️ Also, please build the theme and make the changes in the `material` folder, or they won't be picked up.